### DISCUSSION TITLE
我想知道，为什么管理器里边设计成两个页面？没理解左右滑动的意义在哪里

### DISCUSSION THREAD

**Discussion by weitianxiang19910000** [TeamAmaze/AmazeFileManager#3880] [Q&A] [answered]
我想知道，为什么管理器里边设计成两个页面？没理解左右滑动的意义在哪里

edit: translation: "I would like to know why there are two pages in the manager? I don't understand the meaning of swiping left and right"

**Comment by TranceLove** [comment, marked as answer]
@weitianxiang19910000 We use pages to represent the 2 tabs. This enable users to copy files between different locations easily.

**Comment by weitianxiang19910000** [comment]
非常优秀的设计，很好用，感谢解答

edit: translation: "Very good design, very easy to use, thank you for your answer"
